apps/avifenc.c

```diff
@@ -76,12 +76,27 @@ static void syntaxShort(void)
 static void syntaxLong(void)
 {
     printf("Syntax: avifenc [options] input.[jpg|jpeg|png|y4m] output.avif\n");
-    printf("Options:\n");
+    printf("Standard options:\n");
```

Comment:
The output of `avifenc --longhelp` looks like the following:

```
Syntax: avifenc [options] input.[jpg|jpeg|png|y4m] output.avif
Standard options:
    -h,--help                         : Show short syntax help
    --longhelp                        : Show long syntax help
    -V,--version                      : Show the version number

Basic options:
    -q,--qcolor Q                     : Set quality for color (0-100, where 100 is lossless)
    --qalpha Q                        : Set quality for alpha (0-100, where 100 is lossless)
    -s,--speed S                      : Encoder speed (0-10, slowest-fastest, 'default' or 'd' for codec internal defaults. default speed: 6)

Advanced options:
    -j,--jobs J                       : Number of jobs (worker threads, default: 1. Use "all" to use all available cores)
    ...

aom-specific advanced options:
    1. <key>=<value> applies to both the color (YUV) planes and the alpha plane (if present).
    ...

Version: 0.11.1 (dav1d [dec]:1.2.0-0-g676a864, libgav1 [dec]:0.18.0, aom [enc/dec]:3.6.1, rav1e [enc]:0.6.6 (v0.6.6), svt [enc]:v1.6.0)
libyuv : available (1864)

```

Reply:
Optional: Putting short flags last makes it easier to read for me:
```
--help,-h
--longhelp
--version,-V
```
It has to do with alignment I guess? But the opposite might be more widespread. Increasing the gap between the flags could help as well: `-h / --help` (as in [cwebp](https://github.com/webmproject/libwebp/blob/25d94f473b10882b8bee9288d00539001b692042/examples/cwebp.c#L536)).

Reply:
I looked at the man pages of a few commands such as `cp`, 'mv', `rm`, and `tar` on Mac and Linux. They list the short options before the long options. The short and long options are separated by either a comma, e.g.,

```
       -v, --verbose
              explain what is being done
```

or by a newline, e.g.,

```
       -W
       --no-warn
           Suppress warning messages.
```